ta/pkcs11/src/serializer.h

```diff
@@ -65,6 +68,29 @@ static inline enum pkcs11_rc serialargs_get_u32(struct serialargs *args,
 enum pkcs11_rc serialargs_get_ptr(struct serialargs *args, void **out,
 				  size_t size);
 
+/*
+ * serialargs_alloc_get_one_attribute() - allocate and extract one attribute
+ * @args:	serializing state
+ * @out:	Pointer to the allocated and extracted attribute in *@out
+ *
+ * Returns PKCS11_CKR_OK on success or an error code from enum pkcs11_rc on
```

Comment:
Could maybe remove this line since obvious from function return value type.

Reply:
Yes it's obvious for some of us, in fact the entire description is rather obvious to me.

Reply:
ok